opencog/moses/main/populate_atomspace.h

```diff
@@ -56,6 +57,64 @@ void populate(AtomSpace *as, const CTable &ctable);
  * @return ValuePtr containing values such as FloatValue, LinKValues...etc
  */
 ValuePtr vertex_seq_to_value(const vertex_seq& vseq, id::type_node tnode);
+#define GET_ENUM [](const vertex &ver) { \
+    auto en = get_enum_type(ver); \
+    return ValuePtr(createNode(CONCEPT_NODE, en.getContent())); \
+    }
+		
+template <typename... HandleArgs>
+inline void populate_frm_ctable(CTable _ctable, HandleArgs &&... handleArgs) {
```

Comment:
I don't think you need to worry about `inline`. Did you add it to avoid a compiler error? If not you can just remove it, the gcc is usually smart enough to know when to inline or not.

Reply:
No actually there was no error, I added it since the  func implementation was on the header file. we can remove it

Reply:
Since it is a template I think it is not required. Now when its not a template, and the implementation is in the header you need to add the `static` keyword (though I think `inline` works as well indeed).